src/Components/Shifting/ShiftingBoard.tsx

```diff
@@ -18,7 +18,7 @@ interface boardProps {
 
 const now = moment().format("DD-MM-YYYY:hh:mm:ss");
 
-const renderBoardTitle = (board: string) => board === "APPROVED" ? "AWAITING DESTINATION APPROVED" : board
+const renderBoardTitle = (board: string) => board === "APPROVED" ? "AWAITING DESTINATION APPROVAL" : board
```

Comment:
no idea why APPROVED text alone is handled like this rather than changing the text in constant.